maraboupy/test/test_nnet_extensions.py

```diff
@@ -0,0 +1,149 @@
+
+from maraboupy.MarabouNetworkNNet import *
+
+
+from subprocess import call
+
+property_filename = "./resources/properties/acas_property_4.txt"
+network_filename = "./resources/nnet/acasxu/ACASXU_experimental_v2a_1_9.nnet"
+
+layer = 2
+
+
+def test_write_to_file():
+    output_filename = "./maraboupy/test/ACASXU_experimental_v2a_1_9_output.nnet"
```

Comment:
A temporary file can be used. See the first test in test_query.py for an example. A "tmpdir" argument is added to the test method, which can be used to make a temporary file for writing the nnet file.